docs/visual/e2e-testing/troubleshooting.md

```diff
@@ -95,3 +95,16 @@ capabilities = {
 ```
 
 Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).
+
+
+## cropTo can not crop the target element
+
+If you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:
+
+```
+await browser.execute('/*@visual.snapshot*/', 'Home', { scrollAndStitchScreenshot: true });
+```
+
+### limitation
+
+If the element is scrollable, we are unfortunately not supporting this feature for now.
```

Comment:
```suggestion
If the target element is scrollable, the `scrollAndStitchScreenshot` option isn't supported.
```

Reply:
when the element is scrollable, it's the cropTo option that won't work, I updated the description. Please review.

Reply:
Thanks for clarifying. 

So if the target element isn't scrollable, we recommend to use `scrollAndStitchScreenshot`. And if the target element is scrollable, we also recommend to use `scrollAndStitchScreenshot` because `cropTo` may not work -- is that correct?

Reply:
If the target element isn't scrollable, we recommend to use scrollAndStitchScreenshot.

But if the target element is scrollable, this scrollAndStitchScreenshot and cropTo options won't work out.

Reply:
Got it - thank you!